src/platform/cannonlake/cannonlake.x.in

```diff
@@ -96,6 +96,9 @@ MEMORY
   static_log_entries_seg (!ari) :
         org = LOG_ENTRY_ELF_BASE,
         len = LOG_ENTRY_ELF_SIZE
+  fw_metadata_seg (!ari) :
```

Comment:
wont we need this for all the platforms ?

Reply:
Yes, added